app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  after_save :update_post_counter
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be better if you call **change_post_counter** in the after_save callback because in the current situation, your application will throw an error that update_post_counter doesn't exist 😨